components/tyk-mdcb/README.md

```diff
@@ -0,0 +1,121 @@
+## Tyk Multi Data Centre Bridge (MDCB)
+
+Tyk’s Multi Data Centre Bridge (MDCB) is a separately licensed extension to the Tyk control plane that performs management 
+and synchronisation of logically or geographically distributed clusters of Tyk API Gateways. 
+
+## Introduction
+This chart defines a standalone Tyk MDCB component on a [Kubernetes](https://kubernetes.io/) cluster using the [Helm](https://helm.sh/) package manager.
+
+For typical usage, we recommend using following umbrella charts:
+* For single data centre deployment, please use [tyk-stack](https://github.com/TykTechnologies/tyk-charts/tree/main/tyk-stack)
+* For multi data centre deployment, please use tyk-control-plane (Coming soon!)
+
+[Learn more about different deployment options](https://tyk.io/docs/apim/)
+
+## Prerequisites
+* Kubernetes 1.19+
+* Helm 3+
+* [Tyk Gateway](../tyk-gateway)
+* [Tyk Dashboard](../tyk-dashboard)
+* [Redis](https://tyk.io/docs/planning-for-production/redis/) should already be installed or accessible by the gateway
+*  [MongoDB](https://tyk.io/docs/planning-for-production/database-settings/mongodb/) or [PostgreSQL](https://tyk.io/docs/planning-for-production/database-settings/postgresql/)
+
+
+## Installing the Chart
+
+To install the chart from the Helm repository in namespace `tyk` with the release name `tyk-mdcb`:
+
+```bash
+helm repo add tyk-helm https://helm.tyk.io/public/helm/charts/
+helm show values tyk-helm/tyk-mdcb > values.yaml
+```
+
+Note:
+* Set Redis connection details at `.Values.global.redis`
+* Set MongoDB or connection details either at `.Values.global.mongo` or `.Values.global.postgres`
+* The Tyk MDCB also requires a license to be set at `.Values.mdcb.license`
+
+```bash
+helm install tyk-mdcb tyk-helm/tyk-mdcb -n tyk --create-namespace -f values.yaml
```

Comment:
Need to include `--devel` as it will be beta release.

Reply:
Will it be a beta? Or `v0.0.1`. according to that, I also need to chart version.

Reply:
Can you confirm with carol?

Reply:
updated according to beta release